include/topgg/client.h

```diff
@@ -9,13 +9,7 @@
  */
 
 #pragma once
-
-#include <topgg/topgg.h>
-
-#include <functional>
-#include <vector>
-#include <string>
-#include <map>
+#include "topgg/export.h"
```

Comment:
I feel like this should've been `#include <topgg/export.h>`. Not that big of an issue, but this would've made it consistent!

Reply:
Sure!